doc/config.md

```diff
@@ -0,0 +1,112 @@
+# Configuration format of qem-bot
+
+Configuration,in another way also called metadata of qem-bot is hold in yaml files with one file per "product". Plus there is one special config file called `singlearch.yml` which contains list of packages that exist only on single architecture. 
+
+Example of `singlearch.yaml`:
+
+```yaml
+- packageone
+- packagetwo
+- onlyppcle
+```
+
+If incident has package from this list, job is automaticaly marked as without **aggregate**. With this incident can be approved without existing aggregate jobs.
+Rest yaml files contains definition of **product** and data either or both **aggregate** and **incidents**
```

Comment:
```suggestion
The other yaml files contain the definition of **product** and data either or both **aggregate** and **incidents**
```
or
```suggestion
The other yaml files contain definitions of **product** and data either or both **aggregate** and **incidents**
```
or something else? I don't really understand this sentence. Especially the last part.